### PR TITLE
Provide list of configuration values for Helm Chart

### PR DESCRIPTION
**Pull request by tomkerkhove**
Provide list of configuration values for Helm Chart

Relates to #202

### CODE REVIEW

File: chart/keda/README.md [hunk 1]
```diff
@@ -45,6 +45,21 @@ $ helm install kedacore/keda-edge --devel --set logLevel=debug --namespace keda
...
+| `image.repository`                | Repository which provides the image | `kedacore/keda`      |
+| `image.tag`                       | Tag of image to use | `lastest`            |
+| `image.pullPolicy`                | Policy to pull image | `Always`            |
+| `replicaCount`                    | Amount of replicas to run | `1`            |
+| `customResourceDefinition.create` | Indication to whether or not to create the custom resource definition | `true`            |
+| `rbac.create`                     | Indication to whether or not to use role-based access control | `true`            |
+| `serviceAccount.create`           | Indication to whether or not to a serivce account should be used | `true`            |
+| `serviceAccount.name`             | Name of the service account to use | ``            |
+| `logLevel`                        | Granularity of logs to use | `info`          |
+| `glogLevel`                       | Granularity of logs to use | `2`            |
```
Comment (tomkerkhove) [inline chart/keda/README.md]: There seems to be overlap in terms of log level, do we need both?

Comment (ahmelsayed) [reply chart/keda/README.md]: Unfortunately yes. One controls our log messages, and one controls other dependencies that are using a different logger. We should try to unify them though.

Comment (tomkerkhove) [reply chart/keda/README.md]: Created #238 to track this

File: chart/keda/README.md [hunk 2]
```diff
@@ -45,6 +45,21 @@ $ helm install kedacore/keda-edge --devel --set logLevel=debug --namespace keda
...
+|:----------------------------------|:------------------------------------|:---------------------|
+| `image.repository`                | Repository which provides the image | `kedacore/keda`      |
+| `image.tag`                       | Tag of image to use | `lastest`            |
+| `image.pullPolicy`                | Policy to pull image | `Always`            |
+| `replicaCount`                    | Amount of replicas to run | `1`            |
+| `customResourceDefinition.create` | Indication to whether or not to create the custom resource definition | `true`            |
+| `rbac.create`                     | Indication to whether or not to use role-based access control | `true`            |
+| `serviceAccount.create`           | Indication to whether or not to a serivce account should be used | `true`            |
+| `serviceAccount.name`             | Name of the service account to use | ``            |
+| `logLevel`                        | Granularity of logs to use | `info`          |
```
Comment (tomkerkhove) [inline chart/keda/README.md]:
```suggestion
| `logLevel`                        | Granularity of logs to use | `info`          |
```

Comment (tomkerkhove) [inline chart/keda/README.md]:
```suggestion
| `logLevel`                        | Granularity of logs to use for the metrics adapter | `info`          |
```

Would this description make sense @ahmelsayed ? Now it's a bit vague.

File: chart/keda/README.md (hunk 1 above)
Comment (tomkerkhove) [inline chart/keda/README.md]:
```suggestion
| `glogLevel`                       | Granularity of logs to use for scale controller | `2`            |
```
Would this description make sense @ahmelsayed ? Now it's a bit vague.

File: chart/keda/README.md (hunk 2 above)
Comment (Aarthisk) [reply chart/keda/README.md]: @tomkerkhove the logLevel is the granularity of logs to use for both the metrics adapter and the scale controller- so all Keda.
gLogLevel determines the log of additional components we use- the MetricAdapter boilerplate etc.

So a comment describing that would be useful. Users can start out with setting logLevel =Debug to get more KEDA specific logs. They can make the glogLevel more verbose to get the REST calls made by HPA etc to the metrics adapter. HTH.

Comment (tomkerkhove) [reply chart/keda/README.md]: Thanks for elaborating on this, so basically:
- `logLevel` controls KEDA-specific logs (controller + metrics adapter
- `glogLevel` controls metric adapter core metrics, not specific to KEDA.

Is that correct summary? If so, I'll suggest a better description.

Comment (Aarthisk) [reply chart/keda/README.md]: @tomkerkhove That is correct. And setting the glog verbose level will also pull in the glogs of other dependencies we have (though I primarily see the REST API calls).

Comment (tomkerkhove) [reply chart/keda/README.md]: I've made the changes accordingly, feel free to let me know what you think @Aarthisk

File: chart/keda/README.md (hunk 1 above)
Comment (tomkerkhove) [reply chart/keda/README.md]: See other conversation

**Comment by tomkerkhove** [comment]
And it looks like I've messed up my rebase, sorry for that! I have only changed the README of the Helm chart.

**Comment by tomkerkhove** [comment]
Are we good to go @Aarthisk @yaron2 @ahmelsayed?

**Comment by ahmelsayed** [comment]
yes, though can you rebase on `origin/master`?

**Comment by tomkerkhove** [comment]
The branch is messed up, created a new one and opened PR #265

**Pull request closed without merging**